backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -196,24 +178,20 @@ public ResolvedConceptsResult resolveFilterValues(SelectFilter<?> filter, List<S
 			}
 		}
 
-		return new ResolvedConceptsResult(null, new ResolvedFilterResult(filter.getConnector().getId(), filter.getId(), out), openSearchTerms);
-	}
+		// Not all Searchables are children of Connectors.
+		final ConnectorId connectorId = searchable instanceof Filter asFilter ? asFilter.getConnector().getId() : null;
 
-	@Data
-	@RequiredArgsConstructor(onConstructor_ = {@JsonCreator})
-	public static class AutoCompleteResult {
-		private final List<FrontendValue> values;
-		private final long total;
+		return new ResolvedFilterValues(new ResolvedFilterResult(connectorId, searchable.getId().toString(), out), openSearchTerms);
```

Comment:
Dann packst du hier in ResolvedFilterResult potentiell `connectorId == null` macht das Sinn? Ein Filter ist doch immer einen Connector untergeordnet oder ist hier ein anderer Filter gemeint?

Reply:
ja, das passiert wenn ich direkt auf einer Referenz suche, oder einer Spalte, das ist ja genau die Idee hinter der API Anpassung.